tokio-trace/src/span.rs

```diff
@@ -195,6 +195,20 @@ pub struct Entered<'a> {
     span: &'a Span,
 }
 
+/// A guard representing a span which has been entered and is currently
+/// executing.
+///
+/// When the guard is dropped, the span will be exited.
+///
+/// This is returned by the [`Span::enter`] function.
```

Comment:
I think this is false?

Reply:
whoops, this is what i get for copying the documentation from `Entered`